run-ci.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ev
+
+npm run test-ci
+npm run cobertura
+npm run lint
```

Comment:
@chrux only new dependency is coveralls, the rest are already in package.json devDependencies.

I don't want coveralls in there cause it's only ever used when this is run in CI